app/views/users/edit.html.erb

```diff
@@ -0,0 +1,10 @@
+<% provide(:button_text, 'Save changes') %>
+<div class="container">
+  <div class="row text-center">
+    <div class="offset-lg-4">
+      <h2>Editing user <%= @user.username %> </h2>
```

Comment:
I personally don't like the space between the closing erb tag and closing h2 tag. Not a huge thing but it just doesn't need to be there imo.